api/include/opentelemetry/metrics/meter.h

```diff
@@ -127,7 +127,7 @@ class Meter
    * @param description a brief description of what the Observable UpDownCounter is used for.
    * @param unit the unit of metric values following https://unitsofmeasure.org/ucum.html.
    */
-  virtual nostd::shared_ptr<ObservableInstrument> CreateLongObservableUpDownCounter(
+  virtual nostd::shared_ptr<ObservableInstrument> CreateInt64ObservableUpDownCounter(
```

Comment:
This should be `CreateUInt64ObservableUpDownCounter` as this instrument always reports absolute value, unlike synchronous up-down counter, which reports incremental/delta value which can be positive or negative.

Reply:
Please ignore this comment. API specs is not consistent with examples.